client/influxdb.go

```diff
@@ -145,6 +145,27 @@ func (c *Client) Ping() (time.Duration, string, error) {
 	return time.Since(now), version, nil
 }
 
+func (c *Client) Dump(db string) (*http.Response, error) {
```

Comment:
OK, but then others consuming our library will need to include the io package, right?